lib/src/chrome.dart

```diff
@@ -1,3 +1,244 @@
+        .firstWhere((line) => line.startsWith('DevTools listening'))
+        .timeout(Duration(seconds: 60),
+            onTimeout: () =>
+                throw Exception('Unable to connect to Chrome DevTools.'));
+
+    return _connect(Chrome._(
+      port,
+      ChromeConnection('localhost', port),
+      process: process,
+    ));
+  }
+
+  /// Starts Chrome with the given arguments.
+  ///
+  /// Each url in [urls] will be loaded in a separate tab.
+  static Future<void> start(
+    List<String> urls, {
+    String userDataDir,
+    int remoteDebuggingPort,
+    bool disableBackgroundTimerThrottling = false,
+    bool disableExtensions = false,
+    bool disablePopupBlocking = false,
+    bool bwsi = false,
+    bool noFirstRun = false,
+    bool noDefaultBrowserCheck = false,
+    bool disableDefaultApps = false,
+    bool disableTranslate = false,
+  }) async {
+    await _startProcess(
+      urls,
+      userDataDir: userDataDir,
+      remoteDebuggingPort: remoteDebuggingPort,
+      disableBackgroundTimerThrottling: disableBackgroundTimerThrottling,
+      disableExtensions: disableExtensions,
+      disablePopupBlocking: disablePopupBlocking,
+      bwsi: bwsi,
+      noFirstRun: noFirstRun,
+      noDefaultBrowserCheck: noDefaultBrowserCheck,
+      disableDefaultApps: disableDefaultApps,
+      disableTranslate: disableTranslate,
+    );
+  }
+
+  static Future<Process> _startProcess(
+    List<String> urls, {
+    String userDataDir,
+    int remoteDebuggingPort,
+    bool disableBackgroundTimerThrottling = false,
+    bool disableExtensions = false,
+    bool disablePopupBlocking = false,
+    bool bwsi = false,
+    bool noFirstRun = false,
+    bool noDefaultBrowserCheck = false,
+    bool disableDefaultApps = false,
+    bool disableTranslate = false,
+  }) async {
+    final List<String> args = [];
+    if (userDataDir != null) {
+      args.add('--user-data-dir=$userDataDir');
+    }
+    if (remoteDebuggingPort != null) {
+      args.add('--remote-debugging-port=$remoteDebuggingPort');
+    }
+    if (disableBackgroundTimerThrottling) {
+      args.add('--disable-background-timer-throttling');
+    }
+    if (disableExtensions) {
+      args.add('--disable-extensions');
+    }
+    if (disablePopupBlocking) {
+      args.add('--disable-popup-blocking');
+    }
+    if (bwsi) {
+      args.add('--bwsi');
+    }
+    if (noFirstRun) {
+      args.add('--no-first-run');
+    }
+    if (noDefaultBrowserCheck) {
+      args.add('--no-default-browser-check');
+    }
+    if (disableDefaultApps) {
+      args.add('--disable-default-apps');
+    }
+    if (disableTranslate) {
+      args.add('--disable-translate');
+    }
+    args..addAll(urls);
+
+    final process = await Process.start(_executable, args);
+
+    return process;
+  }
+
+  static Future<Chrome> _connect(Chrome chrome) async {
+    if (_currentCompleter.isCompleted) {
+      throw ChromeError('Only one instance of chrome can be started.');
```

Comment:
I never mentioned this behavior in my code but we should update the start doc comment to make note that only one instance can run at a time.

Reply:
`start` only returns `void` so multiple can actually be started without issue as far as I can tell. 

`startWithPort` returns a `Chrome` instance which exposes the static `connectedInstance` method. The only reason I added this check in initially was because creating multiple `Chrome` instances doesn't make sense with `connectedInstance`. I'm not sure it makes sense to add the `connectedInstance` API to this package. We can always delegate in `package:webdev`.

I also think it's weird that `start` returns `void` while `startWithPort` returns `Chrome`. We should probably align them. I imagine if you just call `start` you'll still want to have a handle on the process to close Chrome.